### PR TITLE
Various cleanup to comply with Inochi2D branding guidelines.

### PR DESCRIPTION
**Pull request by LunaTheFoxgirl**

### CODE REVIEW

**Comment by LunaTheFoxgirl** [comment]
There we go, I had forgotten to save package.d

**Comment by seagetch** [comment]
Sorry to confirmed step by step.
It failed to compile with following error.
```
source/nijigenerate/core/package.d(330,25): Error: undefined identifier `incInitAda`
```

**Comment by LunaTheFoxgirl** [comment]
Ah yeah, didn't catch that.

File: source/nijigenerate/core/package.d [hunk 1]
```diff
@@ -268,7 +268,7 @@ void incOpenWindow() {
     if (!incSettingsGet!bool("DisableCompositor")) SDL_SetHint(SDL_HINT_VIDEO_X11_NET_WM_BYPASS_COMPOSITOR, "0");
 
     version(InBranding) {
-        debug string WIN_TITLE = "nijigenerate "~_("(Debug Mode)");
+        debug string WIN_TincInitAdaITLE = "nijigenerate "~_("(Debug Mode)");
```
Comment (seagetch) [inline source/nijigenerate/core/package.d:271]: Maybe type of WIN_TITLE

**Comment by seagetch** [comment]
LGTM.

**Pull request merged**